execution_workflows/IsoSCM/bin/postprocess.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+
+import sys
+import argparse
+import csv
+
+def parse_args(args=None):
+    Description = "Reformat IsoSCM bed file into the output file of identification"
+    Epilog = "Example usage: python postprcoess.py <FILE_IN> <RUN_IDENTIFICATION> <IDENTIFICATION_FILE_OUT>"
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument("FILE_IN", help="Input IsoSCM output bed file.")
+    parser.add_argument("RUN_MODE", help="Run mode could be 'identification'")
+    parser.add_argument("FILE_OUT", help="Name of output file")
+    return parser.parse_args(args)
+
+
+def parse_input_file(file_in):
+    """
+    This function parses IsoSCM output file to file for
+    identification or relative usage quantification challenge
+    :param file_in: IsoSCM compare step output file to be reformatted to relative usage quantification bed file
+    :return: N/A
+    """
+    outputs = []
+    with open(file_in) as fd:
+        rd = csv.reader(fd, delimiter="\t", quotechar='"')
+        next(fd, None)
+        for row in rd:
+            # obtain proximal PAS
+            chrom = row[2].split(":")[0]
+            # need to do -1 to convert from 1-based gtf file coordinate to 0-based bed file coordinate
+            start = int(row[2].split(":")[1].split("-")[1]) - 1
+            end = start + 1
+            strand = row[8]
+            name = '|'.join([chrom, str(start) + ":" + str(end), strand])
+            output = [chrom, str(start), str(end), name, ".", strand]
+            outputs.append(output)
+
+            # obtain distal PAS
+            # if positive strand, distal PAS is the end of the locus
+            if strand == '+':
+                start = int(row[7].split(":")[1].split("-")[1]) - 1
+            # if negative strand, distal PAS is the start of the locus
+            elif strand == '-':
+                start = int(row[7].split(":")[1].split("-")[0]) - 1
```

Comment:
Request changing `row[7]` to `row[6]` for the explanation given above for the comment on line 43.

Reply:
Should this be row[5]? I see that when strand is -, the upstream segment's coordinates are bigger than downstream segment coordinates. Should we be taking the last coordinate of upstream segment (index 5) for - strand instead of the first coordinate of downstream segment (index 6)? Here is an example of a locus with 3 PAS from the examples I was printing:
<img width="873" alt="Screen Shot 2022-09-29 at 9 36 00 PM" src="https://user-images.githubusercontent.com/25573986/193171328-f5f0e41f-3149-4074-aaa4-6bff10690344.png">

Reply:
@faricazjj no row index 6 is correct for both